pytradfri/device/signal_repeater.py

```diff
@@ -1,16 +1,28 @@
 """Represent a signal repeater."""
-from ..const import ROOT_SIGNAL_REPEATER
+from __future__ import annotations
```

Comment:
Please revert the changes to all files that we don't need to do change otherwise.

Reply:
I have rolled back all changes now